packages/components/src/components/input/input.ts

```diff
@@ -36,8 +36,8 @@ const isFirefox = isChromium ? false : navigator.userAgent.includes('Firefox');
  * @dependency sd-icon
  *
  * @slot label - The input's label. Alternatively, you can use the `label` attribute.
- * @slot prefix - Used to prepend a presentational icon or similar element to the input.
- * @slot suffix - Used to append a presentational icon or similar element to the input.
+ * @slot left - Used to prepend a presentational icon or similar element to the input.
+ * @slot right - Used to append a presentational icon or similar element to the input.
```

Comment:
I think the naming if you're only on Storybook, is confusing. Would suggest to be clearer if we also add an icon.
So, if this slot adds only an icon on the left or right side.. could we rename it to left-icon/right-icon or prefix-icon/suffix-icon?

Reply:
It's not meant only for icons. You can add e. g. `EUR` or `mm` or something, too.

I would have liked prefix and suffix more, but as we already (unfortunately) have `icon-left` and `icon-right` this is the consequence. ^^

Reply:
As this a slot name, it is ok.

Reply:
Can we then modify the text a bit and write `icon, EUR, or a similar element`? I can imagine that others might be confused as well, thinking that only an icon fits there. Without your example with EUR and mm, I wouldn't know what a similar element could be.

Reply:
Then maybe just `text, icons or arbitrary content` to the input?

Reply:
@abudd1094  Can you change the text for left & right as suggested by Mario?